docs/backups-storage.md

```diff
@@ -11,10 +11,16 @@ Configure backup storage for your [backup repositories](backups.md#backup-reposi
      * The region - the location of the bucket
      * S3 credentials such as S3 key and secret to access the storage. These are stored in an encoded form in [Kubernetes Secrets :octicons-link-external-16:](https://kubernetes.io/docs/concepts/configuration/secret/) along with other sensitive information. 
 
+     <i warning>:material-alert: Warning:</i> If you are using an S3 storage in a region located far away from the region of your PostgreSQL cluster deployment, it could lead to the delay of archiving and impossibility to create a new replica/join delayed replica if the primary restarts. 
+
+     When using the default settings, both full and incremental backups may fail because a new WAL file is archived in 60 seconds at the backup start. 
```

Comment:
If we mention some default timeout, we need to give a hint on changing it (otherwise it's not default, but hardcoded).

Reply:
Also, we have a dedicated section later in this document, "Speed-up backups with pgBackRest asynchronous archiving", which is highly relevant to this warning (speed-up WAL archiving). I think, it would be better if we put a really brief warning here, and a link to this section, and move all other info with additional explanations to it